pkg/services/searchV2/extract/testdata/special-datasource-types-info.json

```diff
@@ -0,0 +1,77 @@
+{
+  "id": 221,
+  "title": "special ds",
+  "tags": null,
+  "datasource": [
+    {
+      "uid": "grafana",
```

Comment:
note that the `-- Grafana --` datasource remains - will need to make sure its permissions are handled appropriately (ie. everyone, always has access to it)
<img width="831" alt="image" src="https://user-images.githubusercontent.com/23451458/179734097-7b25b4a5-400a-4cde-bb9c-5fd9263d49e2.png">